attest/core/src/types/pib.rs

```diff
@@ -33,21 +30,16 @@ impl FromBase64 for PlatformInfoBlob {
             return Err(EncodingError::InvalidInputLength);
         }
 
-        let expected_len = src.len() / 4 * 3;
+        let expected_len = base64_buffer_size(src.len());
```

Comment:
This is the inverse calculation; let's leave it as-is for now
```suggestion
        let expected_len = src.len() / 4 * 3;
```